packages/dev/s2-docs/pages/s2/styling.mdx

```diff
@@ -153,164 +166,13 @@ Type scales include: UI, Body, Heading, Title, Detail, and Code. Each scale has
 <InlineAlert variant="notice">
   <Heading>Important Note</Heading>
   <Content>
-    Only use `<Heading>` and `<Text>` inside Spectrum components with predefined styles (e.g., `<Dialog>`, `<MenuItem>`). They are unstyled by default and should not be used standalone. Use HTML elements with the style macro instead.
+    Only use `<Heading>` and `<Text>` inside Spectrum components with predefined styles (e.g., `<Dialog>`, `<MenuItem>`). They are unstyled by default and should not be used standalone. Use HTML elements with the `style` macro instead.
```

Comment:
The typography section should also explain that you're expected to specify the font everywhere, we do not set it at a global level any longer
people have gotten confused on this point
I'll send a link in slack for one of the discussions